state/example/src/main/java/com/expediagroup/streamplatform/streamregistry/state/example/ExampleAgentApp.java

```diff
@@ -53,22 +51,22 @@ EventSender eventSender(ApolloClient apolloClient) {
 
   @Bean
   EventReceiver eventReceiver(
-      @Value("${bootstrapServers}") String bootstrapServers,
-      @Value("${topic}") String topic,
-      @Value("${groupId}") String groupId,
-      @Value("${schemaRegistryUrl}") String schemaRegistryUrl
+    @Value("${bootstrapServers}") String bootstrapServers,
+    @Value("${topic}") String topic,
+    @Value("${groupId}") String groupId,
+    @Value("${schemaRegistryUrl}") String schemaRegistryUrl
   ) {
     KafkaEventReceiver.Config receiverConfig = KafkaEventReceiver.Config.builder()
-        .bootstrapServers(bootstrapServers)
-        .topic(topic)
-        .groupId(groupId)
-        .schemaRegistryUrl(schemaRegistryUrl)
-        .build();
+      .bootstrapServers(bootstrapServers)
+      .topic(topic)
+      .groupId(groupId)
+      .schemaRegistryUrl(schemaRegistryUrl)
+      .build();
     return new KafkaEventReceiver(receiverConfig);
   }
 
   @Bean
   EntityView entityView(EventReceiver eventReceiver) {
-    return new DefaultEntityView(eventReceiver);
+    return EntityViews.defaultEntityView(eventReceiver);
```

Comment:
Minor: Wonder if we should include a comment to show how to make a metered `EntityView`?